modules/security/pam.nix

```diff
@@ -39,17 +39,24 @@ in
 
 {
   options = {
-    security.pam.enableSudoTouchIdAuth = mkEnableOption (lib.mdDoc ''
-      Enable sudo authentication with Touch ID
+    security.pam.enableSudoTouchIdAuth = mkEnableOption "" // {
+      description = lib.mdDoc ''
```

Comment:
What's the context behind these changes? Is `nixpkgs-unstable` no longer evaluating, or is it cosmetic, and are they backwards compatible?

Reply:
It's cosmetic insofar as the previous call generates [an incoherent option description](https://daiderd.com/nix-darwin/manual/index.html#opt-security.pam.enableSudoTouchIdAuth): "Whether to enable Enable sudo authentication with Touch ID" and an additional period after the parenthetical note. Unfortunately `mkEnableOption` is intuitive when you're using it like `mkEnableOption "this thingy"` and very confusing when you want complete paragraphs; `mkEnableOption "this thingy. It's very cool and useful"` feels kind of like performing SQL injection on English text.